std/string.d

```diff
@@ -484,6 +484,20 @@ ptrdiff_t indexOf(Range)(Range s, in dchar c,
     return -1;
 }
 
+///
+unittest
+{
+    string s = "Hello World";
+    auto i = indexOf(s, 'W');
+    assert(i == 6);
```

Comment:
Why not combine these lines to  `assert(s.indexOf('W') == 6);` or even  `assert("Hello World".indexOf('W') == 6);` if you want to get really compact? Same applies to the others lines in this.